### PR TITLE
Community Article - Reissue Rackspace SSL Certificates

### PR DESCRIPTION
**Pull request by BadCrumpany**

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-952e13b1bd/how-to/).

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-005df96965/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-de6c1903c4/how-to/). :bow:

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-2bcc34ec89/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-9333a74b54/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c06e17c6f2/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4f5cd7839c/how-to/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-1d705c998a/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-4e42c17393/how-to/).

**Comment by WilliamLoy** [comment]
@cyrichardson  ready for you :)

**Comment by cyrichardson** [comment]
@WilliamLoy On it.

**Comment by cyrichardson** [comment]
@WilliamLoy Only 1 small nit suggested: 
[reissue-or-revoke-and-replace-rackspace-ssl-certificates_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2852623/reissue-or-revoke-and-replace-rackspace-ssl-certificates_cyrEdits.docx)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-995e91e557/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-601bbb142b/how-to/)

File: content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+last_modified_by: William Loy
+product: general
+product_url: general
+---
+
+This article provides instructions for reissuing or revoking and replacing Rackspace Secure Sockets Layer (SSL) certificates by using the Rackspace Control Panel.
+
+Use the following steps to reissue or revoke and replace the certificates:
+
+1. Log in to [my.rackspace.com](https://my.rackspace.com).
```
Comment (RackerKate) [inline content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md]: Please change to login.rackspace.com in the linked text and actual link.

File: content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+2. Expand the **Network** dropdown menu, then click the **SSL certificates** option.
+
+3. Under the **Configured** tab, select the blue highlighted domain name for which you want to update the SSL certificate.
+
+4. Select **Reissue** or **Revoke & Replace** on the right-hand side of the screen.
+
+    - Symantec&reg; certificates prompt with the option to **Revoke and Replace** the certificate.
```
Comment (RackerKate) [inline content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md]: Suggest "...prompt you with..."

File: content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+        **NOTE:** Your SSL certificate is not automatically revoked on your server. The current SSL certificate remains valid until the new certificate is placed on the intended device.
+
+    - Thawte&reg; certificates prompt with the option to **Reissue** the certificate.
```
Comment (RackerKate) [inline content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md]: Suggest "...prompt you with..."

File: content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+2. Expand the **Network** dropdown menu, then click the **SSL certificates** option.
+
+3. Under the **Configured** tab, select the blue highlighted domain name for which you want to update the SSL certificate.
+
+4. Select **Reissue** or **Revoke & Replace** on the right-hand side of the screen.
+
+    - Symantec&reg; certificates prompt with the option to **Revoke and Replace** the certificate.
+
+        **NOTE:** Your SSL certificate is not automatically revoked on your server. The current SSL certificate remains valid until the new certificate is placed on the intended device.
```
Comment (RackerKate) [inline content/general/reissue-or-revoke-and-replace-rackspace-ssl-certificates.md]: Suggest maybe changing this to **Important** instead.

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-2e75771b3b/how-to/)

**Pull request merged**